ui_framework/src/services/accessibility/combo_box_key_codes.js

```diff
@@ -8,17 +8,17 @@
  */
 
 import {
-  DOWN_KEY_CODE,
-  ENTER_KEY_CODE,
-  ESC_KEY_CODE,
-  TAB_KEY_CODE,
-  UP_KEY_CODE,
+  DOWN,
+  ENTER,
+  ESCAPE as ESC,
```

Comment:
Do you want to change this to just be `ESCAPE` and update the consumers of this service, for consistency?